### DISCUSSION TITLE
GoPro Hero 10 upside down images upload

### DISCUSSION THREAD

**Discussion by kojak357** [gopro/labs#1075] [Q&A]
Hi there

My H10 is turned upside down to fit into a housing. I want to have the images taken to be the correct way up. I have change the orientation into all the settings given but it is still uploading images upside down! How do I let a client look at images that is uploaded upside down? The camera does have the correct Firmware before asking.
I need help please.

**Comment by dnewman-gpsw** [comment]
This doesn't seem to be a Labs question.  Is this a Labs only issue?
